### ISSUE TITLE
Uncaught Error in Yii2 module (travis)

### ISSUE THREAD

**Issue by brezzhnev** [Codeception/Codeception#3726]
#### What are you trying to achieve?

Run acceptance tests on Travis CI (local works fine).
Job log available at: https://travis-ci.org/brezzhnev/Test.Tour-booking-system/jobs/175091310#L463
I'm using basic Yii2 template.

#### What do you get instead?

I see next error

```bash
PHP Fatal error:  Uncaught Error: Call to a member function resetPersistentVars() on null in /home/travis/build/brezzhnev/Test.Tour-booking-system/vendor/codeception/base/src/Codeception/Module/Yii2.php:195
Stack trace:
#0 /home/travis/build/brezzhnev/Test.Tour-booking-system/vendor/codeception/base/src/Codeception/Subscriber/Module.php(68): Codeception\Module\Yii2->_after(Object(Codeception\Test\Cest))
#1 [internal function]: Codeception\Subscriber\Module->after(Object(Codeception\Event\TestEvent), 'test.after', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#2 /home/travis/build/brezzhnev/Test.Tour-booking-system/vendor/symfony/event-dispatcher/EventDispatcher.php(174): call_user_func(Array, Object(Codeception\Event\TestEvent), 'test.after', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#3 /home/travis/build/brezzhnev/Test.Tour-booking-system/vendor/symfony/event-dispatcher/EventDispatcher.php(43): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'test.after', Object(Codeception\E in /home/travis/build/brezzhnev/Test.Tour-booking-system/vendor/codeception/base/src/Codeception/Module/Yii2.php on line 195
```
My acceptance test includes this code

```php
/**
 * Ensure that index page works
 * @param AcceptanceTester $I
 */
public function indexPageWorks(AcceptanceTester $I)
{
    $I->amOnPage(Url::toRoute('/booking/index'));
    $I->see('Bookings');
}
```
### Details

* Codeception version: 2.2.3
* PHP Version: 7.0.13
* Operating System: Ubuntu 16.04 (local), Travis - unknown 
* Installation type: Composer
* Suite configuration:

**acceptance.suite.yml**

```yml
class_name: AcceptanceTester
modules:
    enabled:
        - WebDriver:
            url: 'http://127.0.0.1:8888'
            browser: phantomjs
        - Yii2:
            part: orm
            entryScript: index-test.php
            cleanup: false
```

**.travis.yml**

```yml
language: php

php:
  - 7.0
#  - hhvm

# faster builds on new travis setup not using sudo
sudo: false

# cache vendor dirs
cache:
  directories:
    - $HOME/.composer/cache

before_script:
  - psql -c 'create database "test.tour-booking-system";' -U postgres
  - php yii migrate --interactive=0
  - php -S localhost:8888 -t web > /dev/null 2>&1 &
  - phantomjs --webdriver=4444 >/dev/null 2>&1 &
  - composer install --prefer-source

services:
  - postgresql

install:
  - travis_retry composer self-update && composer --version
  - travis_retry composer global require "fxp/composer-asset-plugin:~1.1.1"
  - travis_retry composer update --dev --prefer-dist --no-interaction
# setup application:
  - |
    sed -i "s/'cookieValidationKey' => ''/'cookieValidationKey' => 'testkey'/" config/web.php

script:
  - |
    vendor/bin/codecept build
    vendor/bin/codecept run
```

**Comment by githubjeka** [comment]
Dubl https://github.com/Codeception/Codeception/issues/3705

**Comment by lostemop** [comment]
You try to use yii2 / vendor / codecept, do not use codecept / codecept / base / codecept

**Comment by lukeholder** [comment]
I am getting this same error on my local machine:

```
➜  craft git:(feature/tests) ✗ ./vendor/bin/codecept run
Codeception PHP Testing Framework v2.2.6
Powered by PHPUnit 5.6.7 by Sebastian Bergmann and contributors.

Acceptance Tests (1) --------------------------------------------------------------------------
E LoginCest: Check open 

Fatal error: Uncaught Error: Call to a member function resetPersistentVars() on null in /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Module/Yii2.php on line 195

Error: Call to a member function resetPersistentVars() on null in /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Module/Yii2.php on line 195

Call Stack:
    0.0010     372144   1. {main}() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/codecept:0
    0.0843    5626064   2. Codeception\Application->run() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/codecept:33
    0.0843    5626064   3. Symfony\Component\Console\Application->run() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Application.php:103
    0.0966    5777080   4. Symfony\Component\Console\Application->doRun() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/console/Application.php:122
    0.0968    5777080   5. Symfony\Component\Console\Application->doRunCommand() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/console/Application.php:191
    0.0968    5777080   6. Symfony\Component\Console\Command\Command->run() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/console/Application.php:829
    0.0974    5779208   7. Codeception\Command\Run->execute() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/console/Command/Command.php:255
    0.1254    6734736   8. Codeception\Command\Run->runSuites() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Command/Run.php:286
    0.1255    6734736   9. Codeception\Codecept->run() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Command/Run.php:359
    0.1298    6739256  10. Codeception\Codecept->runSuite() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Codecept.php:178
    1.0023    8952792  11. Codeception\SuiteManager->run() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Codecept.php:209
    1.0042    8991024  12. Codeception\PHPUnit\Runner->doEnhancedRun() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/SuiteManager.php:162
    1.0044    8999728  13. PHPUnit_Framework_TestSuite->run() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/PHPUnit/Runner.php:98
    1.0045    9000296  14. Codeception\Test\Test->run() /Users/lh/Code/Projects/Craft3/craft/vendor/phpunit/phpunit/src/Framework/TestSuite.php:728
    1.0187    9311008  15. PHPUnit_Framework_TestResult->endTest() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Test/Test.php:112
    1.0187    9311008  16. Codeception\PHPUnit\Listener->endTest() /Users/lh/Code/Projects/Craft3/craft/vendor/phpunit/phpunit/src/Framework/TestResult.php:401
    1.0187    9311360  17. Codeception\PHPUnit\Listener->fire() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/PHPUnit/Listener.php:110
    1.0187    9311360  18. Symfony\Component\EventDispatcher\EventDispatcher->dispatch() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/PHPUnit/Listener.php:124
    1.0187    9311760  19. Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/event-dispatcher/EventDispatcher.php:43
    1.0187    9311760  20. call_user_func:{/Users/lh/Code/Projects/Craft3/craft/vendor/symfony/event-dispatcher/EventDispatcher.php:174}() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/event-dispatcher/EventDispatcher.php:174
    1.0187    9311760  21. Codeception\Subscriber\Module->after() /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/event-dispatcher/EventDispatcher.php:174
    1.0187    9311760  22. Codeception\Module\Yii2->_after() /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Subscriber/Module.php:68




FATAL ERROR. TESTS NOT FINISHED.
Uncaught Error: Call to a member function resetPersistentVars() on null in /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Module/Yii2.php:195
Stack trace:
#0 /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Subscriber/Module.php(68): Codeception\Module\Yii2->_after(Object(Codeception\Test\Cest))
#1 [internal function]: Codeception\Subscriber\Module->after(Object(Codeception\Event\TestEvent), 'test.after', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#2 /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/event-dispatcher/EventDispatcher.php(174): call_user_func(Array, Object(Codeception\Event\TestEvent), 'test.after', Object(Symfony\Component\EventDispatcher\EventDispatcher))
#3 /Users/lh/Code/Projects/Craft3/craft/vendor/symfony/event-dispatcher/EventDispatcher.php(43): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'test.after', Object(Codeception\Event\TestEvent))
#4 /Users/lh/Code/Projects/Craft3/cra 
in /Users/lh/Code/Projects/Craft3/craft/vendor/codeception/codeception/src/Codeception/Module/Yii2.php:195
```

Relevant composer.json:

```
    "require-dev": {
      "codeception/codeception": "*",
      "codeception/verify": "*",
      "codeception/specify": "*",
      "codeception/mockery-module": "*"
    },
```

**Comment by soap** [comment]
I just encountered this problem "Uncaught Error: Call to a member function resetPersistentVars() on null".
Did you solve this problem?

**Comment by kamilmiem** [comment]
Run selenium and WebDriver ;)

**Comment by Naktibalda** [comment]
Fixed by #3876
